packages/lesswrong/lib/collections/posts/schema.tsx

```diff
@@ -1016,6 +1017,34 @@ const schema: SchemaType<"Posts"> = {
     optional: true,
     hidden: true,
   },
+  
+  digestNum: resolverOnlyField({
+    graphQLtype: 'Int',
+    type: Number,
+    canRead: ['guests'],
+    resolver: async (post: DbPost, _, context: ResolverContext) => {
+      // only relevant to the EA Forum
+      if (!isEAForum) return null
+      return context.repos.posts.getDigestNumForPost(post._id)
+    },
```

Comment:
This is not my favorite thing but I didn't notice any slowness from adding it, though maybe I should try harder to use a loader